src/sidebar/components/test/search-status-bar-test.js

```diff
@@ -26,6 +26,8 @@ describe('SearchStatusBar', () => {
       clearDirectLinkedGroupFetchFailed: sinon.stub(),
```

Comment:
from `selectTab` to `clearSelection,` these actions appear to not be used in the test or required for 11 tests to pass.

Reply:
Ah yes I think those may have been leftover from a previous refactor. Good catch thanks!